lib/nightwatch-handler.js

```diff
@@ -21,6 +22,7 @@ module.exports = function (options) {
     distPath: distPath,
     indexFileName: indexFileName,
     mocksPort: mocksPort,
+    mocksHttpsPort: mocksHttpsPort,
```

Comment:
if we are either starting http or https (but not both) we don't need a `mocksHttpsPort` or am I missing something?